### ISSUE TITLE
elegant: convert string to float: (Sdelta0)

### ISSUE THREAD

**Issue by robnagler** [radiasoft/sirepo#863]
I was poking around the logs and I happened to see this (multiple times) on alpha:

```
ValueError: could not convert string to float: (Sdelta0)
[06/Sep/2017:15:43:41 +0000] "POST /run-simulation HTTP/1.1" 500 0 "https://alpha.sirepo.com/elegant" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:55.0) Gecko/20100101 Firefox/55.0"
```

**Comment by moellep** [comment]
I was causing these errors by importing an elegant example which has RPN values in bunched_beam fields. This will be fixed shortly along with ticket #716

**Comment by robnagler** [comment]
Sorry about that. I didn't check the IP. Just happened to notice...
